apps/cms-server/modules/asset/ui/src/index.js

```diff
@@ -66,13 +66,44 @@ function adjustMenu() {
     closeButtonSpan.textContent = 'Menu tonen';
   }
 
+  function trapFocus(element) {
+    const focusableElements = element.querySelectorAll('a, button, input, textarea, select, details, [tabindex]:not([tabindex="-1"])');
+    const firstFocusableElement = focusableElements[0];
+    const lastFocusableElement = focusableElements[focusableElements.length - 1];
+  
+    element.addEventListener('keydown', (event) => {
+      const isTabPressed = (event.key === 'Tab' || event.keyCode === 9);
+  
+      if (!isTabPressed) {
+        return;
+      }
+  
+      if (event.shiftKey) {
+        if (document.activeElement === firstFocusableElement) {
+          lastFocusableElement.focus();
+          event.preventDefault();
+        }
+      } else {
+        if (document.activeElement === lastFocusableElement) {
+          firstFocusableElement.focus();
+          event.preventDefault();
+        }
+      }
+    });
+  }
+  
+
```

Comment:
We hebben ook een soortgelijke `trapFocus` in `packages/document-map/src/document-map.tsx`. Is er een mogelijkheid om die los te trekken en hier te hergebruiken?

Reply:
Ik weet eigenlijk niet of dat zo makkelijk te doen is. 1 is onderdeel van de componenten, terwijl die andere in het cms ingeladen wordt. Zijn 2 verschillende plekken dan toch?

Reply:
Besproken dat we hier nu een ticket van maken in TL, deze wordt nu gemerged.